### ISSUE TITLE
Make AvizoWindow never having focus

### ISSUE THREAD

**Issue by misterdanb** [misterdanb/avizo#29]
Is it possible to make Gtk/Gdk Tell the compositor that the `AvizoWindow` must Not bei focussed, so all Inputs  are captured by the window that was focussed before?

**Comment by jirutka** [comment]
This is not happening on my system with Sway. When I open a terminal, press the volume up button, Avizo is shown, now when I type it goes to the terminal.

The following line should disable the input capturing: [src/avizo_service.vala:254](https://github.com/misterdanb/avizo/blob/526d7bfa2cef4c07ec3b4c1d9aedf7e3421cc95a/src/avizo_service.vala#L254):
```vala
GtkLayerShell.set_keyboard_interactivity(window, false);
```

**Comment by misterdanb** [comment]
Hmm, I tested this yesterday, and if you explicitly click on the window the other window loses its focus.

**Comment by jirutka** [comment]
I just tried to click on the notification window and it still doesn’t get focus.

Does `y-offset` work for you? What version of gdk-layer-shell do you have?

**Comment by misterdanb** [comment]
Yes, the y-offset was working fine! I'm using the wrong device right now, so I will check the layer shell version tomorrow.

**Comment by jirutka** [comment]
This means that it has to be rendered via Wayland and Layer Shell. I dunno what may be the cause, maybe a bug in some old version?
